test/tests.bats

```diff
@@ -50,7 +52,7 @@ setup_file() {
   docker cp "${PRIVATE_CONFIG}/sieve/dovecot.sieve" mail:/var/mail/localhost.localdomain/user1/.dovecot.sieve
 
   # this relies on the checksum file beeing updated after all changes have been applied
-  wait_for_changes_to_be_detected_in_container mail
+  wait_until_change_detection_event_completes mail
```

Comment:
Attempt to fix transient failure with `nc` mail lines. Postfix is presumably unreachable..?

```suggestion
  wait_until_change_detection_event_completes mail
  wait_for_service mail postfix
```

This seems unlikely though if `wait_for_smtp_port_in_container mail` is working.. It could have something to do with the clamav wait that follows?

Reply:
This seems to be the change that caused the transient `nc` failures show up. I hope #2811 can do something about it.

Reply:
It could, but I think it's more likely that it's helped surfaced awareness of an issue we've had for some time (_possibly related to the cause that required some tests to be disabled_). I am inclined to believe ClamAV is still the actual problem.